src/components/Header/Header.js

```diff
@@ -39,18 +39,18 @@ class Header extends Component {
     }
     return (
       <header style = { userAuth ? darkTheme: lightTheme } >
-      <Link to ='/auth/login'><img src = { userAuth ? '../../img/logo-white.png' : '../../img/logo-dark.png'} className='logo' alt='apiko' /></Link>
+      <Link to = '/auth/login'><img src = { userAuth ? '../../img/logo-white.png' : '../../img/logo-dark.png'} className = 'logo' alt = 'apiko' /></Link>
```

Comment:
suggest to split this line as it's hard to understand it:
```
<Link to = '/auth/login'>
  <img
    src = {
      userAuth ?
      '../../img/logo-white.png'
      : '../../img/logo-dark.png'
    }
    className = 'logo' 
    alt = 'apiko'
  />
</Link>
```
if you move expression to get src into separate const (even inside render)  it would be prettier:
```
//somewhere above in render
const logoSrc = userAuth ?
      '../../img/logo-white.png'
      : '../../img/logo-dark.png'

//and now
<Link to = '/auth/login'>
  <img
    src = {logoSrc}
    className = 'logo' 
    alt = 'apiko'
  />
</Link>
```
by the way, adding spaces around = was not neccesary, usually in templates it goes without them. No need to turn it back, just a note